### DISCUSSION TITLE
How to set stress loading method instead of strain loading method in damask3？

### DISCUSSION THREAD

**Discussion by HU-8969** [damask-multiphysics/DAMASK#52] [Q&A]
May I know how to set my load file if I want to set the stress loading method and then get the stress-strain curve instead of the strain loading method?

**Comment by MarDiehl** [comment]
have a look at https://damask-multiphysics.org/documentation/file_formats/grid_solver.html#load-case

**Reply by HU-8969** [reply]
Thank you very much.May I know how should I set the code if I want to get the maximum shear strain curve of .hdf5 file using Vscode.

**Reply by MarDiehl** [reply]
pls. do not ask unspecific questions and follow https://damask-multiphysics.org/support/helpdesk.html
